fast/driver/driver.c

```diff
@@ -1,14 +1,14 @@
 /*	fcc/fm2/fpc 
-	Driver for fast ACK compilers.
+ Driver for fast ACK compilers.
```

Comment:
I'm not sure this has worked for decades. It's certainly not used by anything modern.

Reply:
Actually i did not update or create the build.lua yet, but it has been updated and it can be used  as a front C compiler to build EM targets. I still need to update it a bit more though...